test/strings.jl

```diff
@@ -1447,3 +1447,6 @@ let s = "ba\0d"
     @test_throws ArgumentError Base.unsafe_convert(Cstring, s)
     @test_throws ArgumentError Base.unsafe_convert(Cwstring, wstring(s))
 end
+
+
+
```

Comment:
Not sure how the blank lines got in, will remove... I've been doing just that sort of testing, as part of performance, I'd posted it as a Gist last week... I'll remove the `@time` and looping from it, clean it up, add `@test` macros, and add it...  Thanks!